crates/service/src/routes/request_handler.rs

```diff
@@ -1,53 +1,58 @@
 // Copyright 2023-, Edge & Node, GraphOps, and Semiotic Labs.
 // SPDX-License-Identifier: Apache-2.0
 
-use std::sync::Arc;
-
-use crate::{error::IndexerServiceError, middleware::Allocation};
+use crate::{
+    error::SubgraphServiceError, middleware::AttestationInput, service::SubgraphServiceState,
+};
 use axum::{
     extract::{Path, State},
+    http::{HeaderValue, Response},
     response::IntoResponse,
-    Extension,
 };
-use reqwest::StatusCode;
+use reqwest::header::CONTENT_TYPE;
 use thegraph_core::DeploymentId;
 use tracing::trace;
 
-use crate::service::{AttestationOutput, IndexerServiceResponse, IndexerServiceState};
-
 pub async fn request_handler(
-    Path(manifest_id): Path<DeploymentId>,
-    Extension(Allocation(allocation_id)): Extension<Allocation>,
-    State(state): State<Arc<IndexerServiceState>>,
+    Path(deployment): Path<DeploymentId>,
+    State(state): State<SubgraphServiceState>,
     req: String,
-) -> Result<impl IntoResponse, IndexerServiceError> {
-    trace!("Handling request for deployment `{manifest_id}`");
+) -> Result<impl IntoResponse, SubgraphServiceError> {
+    trace!("Handling request for deployment `{deployment}`");
 
-    // Check if we have an attestation signer for the allocation the receipt was created for
-    let signer = state
-        .attestation_signers
-        .borrow()
-        .get(&allocation_id)
-        .cloned()
-        .ok_or_else(|| (IndexerServiceError::NoSignerForAllocation(allocation_id)))?;
+    let deployment_url = state
+        .graph_node_query_base_url
+        .join(&format!("subgraphs/id/{deployment}"))
+        .map_err(|_| SubgraphServiceError::InvalidDeployment(deployment))?;
 
     let response = state
-        .service_impl
-        .process_request(manifest_id, &req)
+        .graph_node_client
+        .post(deployment_url)
+        .body(req.clone())
+        .header(CONTENT_TYPE, HeaderValue::from_static("application/json"))
+        .send()
         .await
-        .map_err(IndexerServiceError::ProcessingError)?;
+        .map_err(SubgraphServiceError::QueryForwardingError)?;
 
-    let res = response
-        .as_str()
-        .map_err(|_| IndexerServiceError::FailedToSignAttestation)?;
+    let attestable = response
+        .headers()
+        .get("graph-attestable")
```

Comment:
It might be good to make this a constant, to avoid typos. (I have a hard time writing "attestable" sometimes)
```rust
const HEADER_GRAPH_ATTESTABLE: &str = "graph-attestable";

```